### PR TITLE
HDF5-External-Filter-Plugins: New port

### PR DESCRIPTION
**Pull request by eborisch**
New port.

### CODE REVIEW

File: science/HDF5-External-Filter-Plugins/Portfile [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    }
+}
+
+notes \
+"To enable, use (with H5Pset_filter()):\n filter_id = 307   for bzip2\n\
+filter_id = 32004 for lz4\n\
+filter_id = 32008 for bitshuffle\n\n\
+To enable use from Matlab, or other tools expecting the default HDF5 plugin\
+path, create this symbolic link:\n \
+ /usr/local/hdf5/lib/plugin -> ${prefix}/lib/hdf5"
```
Comment (mf2k) [inline science/HDF5-External-Filter-Plugins/Portfile]: Just put the proper ln command here:

`ln -s /usr/local/hdf5/lib/plugin ${prefix}/lib/hdf5`

File: science/HDF5-External-Filter-Plugins/Portfile [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    rmd160  6724d20763725fe59d04329d5069e2943eec99cb \
+    sha256  3da58e714b94d2e1bffbf52a6b64952733874ef383c23a1ad08cd01cc69f28f9 \
+    size    943586
+
+variant optimized {
+    # Bitshuffle code has sse and AVX implementations
+    configure.cxxflags-append   -march=native
+}
+
+depends_lib         port:hdf5 \
```
Comment (mf2k) [inline science/HDF5-External-Filter-Plugins/Portfile]: When using portgroups, it best practice to use `depends_lib-append` to not overwrite portgroup dependencies.

File: science/HDF5-External-Filter-Plugins/Portfile [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+github.setup        nexusformat HDF5-External-Filter-Plugins 0.1.0 v
+categories          science
+platforms           darwin
+license             BSD MIT
+maintainers         {@eborisch eborisch} openmaintainer
+description         Provides dynamically loadable (compression) filters for HDF5
+long_description    ${description}. Filters for bitshuffle, bzip2, and lz4 are \
+                    included. See also \
+                    http://www.hdfgroup.org/services/filters.html for \
+                    more information.
```
Comment (mf2k) [inline science/HDF5-External-Filter-Plugins/Portfile]: Maybe just set the `homepage` to the above link and remove it from the long_description entirely.

File: science/HDF5-External-Filter-Plugins/Portfile (hunk 1 above)
Comment (eborisch) [reply science/HDF5-External-Filter-Plugins/Portfile]: I debated doing just that. I'm fine either way.

File: science/HDF5-External-Filter-Plugins/Portfile (hunk 2 above)
Comment (eborisch) [reply science/HDF5-External-Filter-Plugins/Portfile]: Makes no difference here, but, as you say, it is best practice. Will do.

File: science/HDF5-External-Filter-Plugins/Portfile (hunk 3 above)
Comment (eborisch) [reply science/HDF5-External-Filter-Plugins/Portfile]: This URL is more of general information for all HDF5 filters, while the (github) homepage is for this particular set of filters. Planning to leave this way.

**Pull request merged**